kubernetes/container-azm-ms-agentconfig.yaml

```diff
@@ -20,6 +20,8 @@ data:
           # If you want to continue to disable kube-system,gatekeeper-system log collection keep the namespaces in the following setting and add any other namespace you want to disable log collection to the array.
           # In the absense of this configmap, default value for exclude_namespaces = ["kube-system","gatekeeper-system"]
           exclude_namespaces = ["kube-system","gatekeeper-system"]
+          # If you want to collect logs from system pods that have been excluded using exclude_namespaces setting, add them to the following setting. Provide daemonset or deployment name of the system pod. Eg adding coredns to list can capture logs from pods with name like coredns-77sd7 or coredns-65cbf4f7bc-jcn6z. NOTE: this setting is only for system pods
+          # include_system_pods = ["coredns"]
```

Comment:
reading the comment,  "include_system_pods" works on excluded system namespaces, is that right? if its case, what's behavior when the system namespace included and also pod names in "include_system_pods"?  I was thinking the system_pods filter works on included system namespaces and collects only the logs for the specified system pods filter. lets discuss more on the behavior we should enable it.

Reply:
if a namespace is already included then this is a noop. include_system_pods only works if the system pod is in the exlude namespace list.